srcpkgs/latte-dock/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'latte-dock'
 pkgname=latte-dock
-version=0.9.4
+version=0.9.8.1
 revision=1
 build_style=cmake
 configure_args="-DBUILD_TESTING=OFF -DENABLE_MAKE_UNIQUE=OFF"
-hostmakedepends="extra-cmake-modules kcoreaddons pkg-config qt5-qmake qt5-host-tools"
-makedepends="knewstuff-devel libksysguard-devel libSM-devel plasma-framework-devel"
+hostmakedepends="extra-cmake-modules kcoreaddons pkg-config qt5-qmake qt5-host-tools gettext"
+makedepends="knewstuff-devel libksysguard-devel libSM-devel plasma-framework-devel gettext"
```

Comment:
Why have gettext in both host- and makedepends? Usually you either need it only in host for the tools or you need -devel in makedepends for the headers.

Reply:
Whoops, marked it resolved before I hit comment. In either case, it built locally (and now in Travis) with your suggested change. I rebased onto master and pushed. Thanks!